### PR TITLE
Screening results fix

### PR DESCRIPTION
**Pull request by Buhle79**

### CODE REVIEW

File: base/actions/actions.py [hunk 1]
```diff
@@ -745,43 +745,55 @@ async def submit(
...
-                                location, 2
-                            )
-
-                            if longitude and latitude:
-                                querystring = urlencode(
-                                    {"longitude": longitude, "latitude": latitude}
+                            location = None
+                            try:
+                                location = json_resp["location"]
+                            except Exception:
```
Comment (erikh360) [inline base/actions/actions.py]: We might want to catch only KeyError here

File: base/actions/actions.py [hunk 2]
```diff
@@ -745,43 +745,55 @@ async def submit(
...
-                                        SlotSet("tbcheck_id", tbcheck_id),
-                                    ]
+                                if longitude and latitude:
+                                    querystring = urlencode(
+                                        {"longitude": longitude, "latitude": latitude}
+                                    )
+
+                                    clinic_url = urljoin(
+                                        config.HEALTHCONNECT_URL,
+                                        "/v1/clinic_finder" f"?{querystring}",
```
Comment (erikh360) [inline base/actions/actions.py]: This line can be one string:
`f"/v1/clinic_finder?{querystring}",`

**Review by erikh360 — COMMENTED** [review]
few small comments

**Comment by Buhle79** [comment]
@erikh360 Please relook

File: base/actions/actions.py (hunk 1 above)
Comment (erikh360) [inline base/actions/actions.py]: I think you updated the wrong exception

**Review by erikh360 — APPROVED** [review]

**Pull request merged**